lib/api/errors/unwritable_property.rb

```diff
@@ -36,13 +36,24 @@ def initialize(invalid_attributes)
         fail ArgumentError, 'UnwritableProperty error must contain at least one invalid attribute!' if attributes.empty?
 
         if attributes.length == 1
-          message = I18n.t('api_v3.errors.writing_read_only_attributes')
+          message = case attributes.first
+                    when 'estimated_hours'
+                      I18n.t('api_v3.errors.validation.estimated_hours')
+                    when 'done_ratio'
+                      I18n.t('api_v3.errors.validation.done_ratio')
+                    else
+                      I18n.t('api_v3.errors.writing_read_only_attributes')
+                    end
```

Comment:
Probably less likely to becoming a long list:

<pre>
message = if attributes.length == 1
  begin
    I18n.t("api_v3.errors.validation.#{attributes.first}", raise: true)
  rescue I18n::MissingTranslationData
    I18n.t('api_v3.errors.writing_read_only_attributes')
  end
 else
  I18n.t('api_v3.errors.multiple_errors')
 end
</pre>

and probably put that part in its own method.